api/server/router/local/image.go

```diff
@@ -56,13 +61,18 @@ func (s *router) postCommit(ctx context.Context, w http.ResponseWriter, r *http.
 		Config:  c,
 	}
 
-	imgID, err := builder.Commit(cname, s.daemon, commitCfg)
+	container, err := s.daemon.Get(cname)
```

Comment:
Using `Get` here is sorta violate our "weak" agreement about separation API from daemon. Maybe we can move it to `Commit`, so API won't be dependent on `Container` struct?